src/app/helpers/Common.ts

```diff
@@ -0,0 +1,10 @@
+export class Common {
```

Comment:
Is it really needed?

Reply:
Yes. It has some functionality, which is used shared between many components.